src/main/resources/MSF4J/dto.mustache

```diff
@@ -1,64 +0,0 @@
-package {{package}};
```

Comment:
This template is not used for code generation for DTO. The used file is pojo.mustache